Slim/Plugin/Podcast/Plugin.pm

```diff
@@ -69,6 +73,23 @@ sub initPlugin {
 		func   => \&trackInfoMenu,
 	) );
 	
+	foreach ( @{$prefs->get('feeds')} ) {
+		my $url = $_->{value};
+
+		Slim::Networking::SimpleAsyncHTTP->new(
+			sub { 
+				eval {
+					my $xml = XMLin(shift->content);
+					my $image = $xml->{channel}->{image}->{url} || $xml->{channel}->{'itunes:image'}->{href};
+					$cache->set("podcast-$url", $image, '30days') if $image;
```

Comment:
Did you want to cache `remote_image_$url` here?

Reply:
No because that's just for RSS itself. It seems (see above) that different episode $url (rarely) have different images